dask/dataframe/io/parquet.py

```diff
@@ -229,8 +241,38 @@ def to_parquet(path, df, compression=None, write_index=None, has_nulls=None,
                                            fixed_text=fixed_text,
                                            object_encoding=object_encoding)
 
+    if append:
+        pf = fastparquet.api.ParquetFile(path, open_with=myopen)
+        if pf.file_scheme != 'hive':
+            raise ValueError('Requested file scheme is hive, '
+                             'but existing file scheme is not.')
+
+        if len(set(pf.columns) ^ set(df._meta.columns)):
+            raise ValueError('Appended columns not the same.\n'
+                             'New: {} | Previous: {}'
+                             .format(pf.columns, list(df._meta.columns)))
+        else:
+            df = df[pf.columns]
+
+        fmd = pf.fmd
```

Comment:
Do I need to check complete schema (`fmd.schema != pf.fmd.schema`) or is it enough to only check the column names and dtypes? Namely I left the option that columns don't need to be sorted correctly, and reorder them if their naming and dtypes are correct.